src/scenarios/shared/runner.py

```diff
@@ -339,6 +341,69 @@ def run(self):
             ]
             RunCommand(cmdline, verbose=True).run()
 
+            # Get animation values
+            getLogger().info("Getting animation values")
+            cmdline = [
+                adb.stdout.strip(),
+                'shell', 'settings', 'get', 'global', 'window_animation_scale'
+            ]
+            window_animation_scale_cmd = RunCommand(cmdline, verbose=True)
+            window_animation_scale_cmd.run()
+            cmdline = [
+                adb.stdout.strip(),
+                'shell', 'settings', 'get', 'global', 'transition_animation_scale'
+            ]
+            transition_animation_scale_cmd = RunCommand(cmdline, verbose=True)
+            transition_animation_scale_cmd.run()
+            cmdline = [
+                adb.stdout.strip(),
+                'shell', 'settings', 'get', 'global', 'animator_duration_scale'
+            ]
+            animator_duration_scale_cmd = RunCommand(cmdline, verbose=True)
+            animator_duration_scale_cmd.run()
+            getLogger().info(f"Retrieved values window {window_animation_scale_cmd.stdout.strip()}, transition {transition_animation_scale_cmd.stdout.strip()}, animator {animator_duration_scale_cmd.stdout.strip()}")
+
+            # Make sure animations are set to 1 or disabled
+            getLogger().info("Setting animation values")
+            if(self.animationsdisabled):
+                animationValue = 0
+            else:
+                animationValue = 1
+            cmdline = [
+                adb.stdout.strip(),
+                'shell', 'settings', 'put', 'global', 'window_animation_scale', str(animationValue)
+            ]
+            RunCommand(cmdline, verbose=True).run()
+            cmdline = [
+                adb.stdout.strip(),
+                'shell', 'settings', 'put', 'global', 'transition_animation_scale', str(animationValue)
+            ]
+            RunCommand(cmdline, verbose=True).run()
+            cmdline = [
+                adb.stdout.strip(),
+                'shell', 'settings', 'put', 'global', 'animator_duration_scale', str(animationValue)
+            ]
+            RunCommand(cmdline, verbose=True).run()
+
+            # Check for success
+            getLogger().info("Getting animation values to verify it worked")
+            cmdline = [
+                adb.stdout.strip(),
+                'shell', 'settings', 'get', 'global', 'window_animation_scale'
+            ]
+            RunCommand(cmdline, verbose=True).run()
+            cmdline = [
+                adb.stdout.strip(),
+                'shell', 'settings', 'get', 'global', 'transition_animation_scale'
+            ]
+            RunCommand(cmdline, verbose=True).run()
+            cmdline = [
+                adb.stdout.strip(),
+                'shell', 'settings', 'get', 'global', 'animator_duration_scale'
+            ]
+            RunCommand(cmdline, verbose=True).run()
```

Comment:
It doesn't look like we do any checking here of the values. Is the goal to just have them printed out to the console so we can see? I feel like it would be better to fail the tests if we are unable to get them configured properly.

Reply:
That is a better idea, let me see what I can do for getting that implemented.

Reply:
I had to reboot in my case, because I didn't know what the original values were.

If you get can get the values, and restore them at the end, that would also work.

Reply:
It's also probably worthwhile to turn these off for other startup tests. I don't see how you'd notice a 10ms regression in the Mono runtime, if it has the splash animation on.